dpdispatcher/contexts/ssh_context.py

```diff
@@ -141,9 +143,13 @@ def _setup_ssh(self):
         #                 )
         # assert(self.ssh.get_transport().is_active())
         # transport = self.ssh.get_transport()
-        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        sock.settimeout(self.timeout)
-        sock.connect((self.hostname, self.port))
+        if self.proxy_command is not None:
+            sock = paramiko.ProxyCommand(self.proxy_command)
+            sock.settimeout(self.timeout)
+        else:
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            sock.settimeout(self.timeout)
+            sock.connect((self.hostname, self.port))
 
         # Make a Paramiko Transport object using the socket
         ts = paramiko.Transport(sock)
```

Comment:
Could you ignore this line before https://github.com/python/typeshed/pull/12697 is accepted?